src/app/pages/assistant/assistant.component.scss

```
@@ -0,0 +1,365 @@
+  transition: background 0.15s, box-shadow 0.15s;
+  text-align: start;
+  width: 100%;
+
+  &:hover {
+    background: var(--hover-bg, rgba(0,0,0,0.03));
+    box-shadow: 0 2px 12px rgba(0,0,0,0.08);
+  }
+}
+
+.result-card-icon,
+.result-card ::ng-deep img {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)